components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/dao/ApiMgtDAO.java

```diff
@@ -8441,6 +8441,42 @@ public String getExternalWorkflowReferenceByApplicationID(int appID) throws APIM
         return workflowExtRef;
     }
 
+    /**
+     * Get external workflow reference by internal workflow reference and workflow type
+     * @param internalRef Internal reference of the workflow
+     * @param workflowType Workflow type of the workflow
+     * @return External workflow reference for the given internal reference and workflow type if present. Null otherwise
+     * @throws APIManagementException
+     */
```

Comment:
Javadoc comments are not properly formatted. Shall we fix those?

Reply:
Fixed in - https://github.com/wso2/carbon-apimgt/pull/11699